pkg/manifest/build.go

```diff
@@ -149,6 +156,33 @@ func (p *BuildrootFromPackages) getSELinuxLabels() map[string]string {
 	return labels
 }
 
+// maybeAddExperimentalContainerBootstrap will return a container buildroot
+// if if the "IMAGE_BUILDER_EXPERIMENTAL=bootstrap=<container-ref>" is
```

Comment:
typo 'if if'

Reply:
Thank you! Fixed now (and while reading this again expanded about what a bootstrap container actually needs).